migrations/versions/0419_add_forced_pass_template.py

```diff
@@ -0,0 +1,65 @@
+"""empty message
+
+Revision ID: 0419_add_forced_pass_template
+Revises: 0327_add_password_expired
+Create Date: 2022-04-19 13:00:00
+
+"""
+
+import uuid
+
+# revision identifiers, used by Alembic.
+from datetime import datetime
+
+from alembic import op
+
+from app.encryption import hashpw
+
+revision = "0419_add_forced_pass_template"
+down_revision = "0327_add_password_expired"
+
+
+user_id = "6af522d0-2915-4e52-83a3-3690455a5fe6"
+service_id = "d6aa2c68-a2d9-4437-ab19-3ae8eb202553"
+
+
+def upgrade():
+    password = hashpw(str(uuid.uuid4()))
```

Comment:
What is this used for?

Reply:
this was a mistake

Reply:
i added the template history table as well if you can review. 🙇🏽‍♀️